packages/react-integration/demo-app-ts/src/components/demos/TooltipDemo/TooltipDemo.tsx

```diff
@@ -21,7 +21,7 @@ export class TooltipDemo extends Component {
     return (
       <div>
         <Tooltip content={this.myTooltipProps.content}>{this.myTooltipProps.children}</Tooltip>
-        <Tooltip content="test deprecated props" tippyProps={{ duration: 0, delay: 0 }} isAppLauncher>
+        <Tooltip content="test deprecated props">
           <button>Trigger</button>
         </Tooltip>
```

Comment:
This tooltip should be able to be removed completely, as the test file doesn't look to be testing anything with it.